src/backend/distributed/sql/citus--10.1-1--10.2-1.sql

```diff
@@ -18,7 +18,9 @@ ALTER TABLE pg_catalog.pg_dist_placement ADD CONSTRAINT placement_shardid_groupi
 #include "udfs/citus_internal_update_relation_colocation/10.2-1.sql";
 #include "udfs/create_time_partitions/10.2-1.sql"
 #include "udfs/drop_old_time_partitions/10.2-1.sql"
+#include "udfs/fix_partition_shard_index_names/10.2-1.sql"
 #include "udfs/get_missing_time_partition_ranges/10.2-1.sql"
+#include "udfs/worker_fix_partition_shard_index_names/10.2-1.sql"
```

Comment:
we can no longer change the citus--10.1-1--10.2-1 upgrade script since it was released

Reply:
Right, I will fix the sql scripts and add citus--10.2-1--10.2-2